src/core_codemods/use_defused_xml.py

```diff
@@ -0,0 +1,97 @@
+import libcst as cst
+from libcst.codemod.visitors import AddImportsVisitor, RemoveImportsVisitor
+
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import BaseCodemod
+from codemodder.codemods.imported_call_modifier import ImportedCallModifier
+
+
+class DefusedXmlModifier(ImportedCallModifier):
+    def update_attribute(self, true_name, _, updated_node, new_args):
+        import_name = self.matching_functions[true_name]
+        AddImportsVisitor.add_needed_import(self.context, import_name)
+        return updated_node.with_changes(
+            args=new_args,
+            func=cst.Attribute(
+                value=cst.parse_expression(import_name),
+                attr=cst.Name(value=true_name.split(".")[-1]),
+            ),
+        )
+
+    def update_simple_name(self, true_name, _, updated_node, new_args):
+        import_name = self.matching_functions[true_name]
+        AddImportsVisitor.add_needed_import(self.context, import_name)
+        return updated_node.with_changes(
+            args=new_args,
+            func=cst.Attribute(
+                value=cst.parse_expression(import_name),
+                attr=cst.Name(value=true_name.split(".")[-1]),
+            ),
+        )
+
+
+ETREE_METHODS = ["parse", "fromstring", "iterparse", "XMLParser"]
+SAX_METHODS = ["parse", "make_parser", "parseString"]
+DOM_METHODS = ["parse", "parseString"]
+# TODO: add expat methods?
+
+
+class UseDefusedXml(BaseCodemod):
+    NAME = "use-defused-xml"
+    SUMMARY = "Use defusedxml for parsing XML"
```

Comment:
```suggestion
    SUMMARY = "Use `defusedxml` for Parsing XML"
```
since it's the title, right?